.github/workflows/release.yaml

```
@@ -0,0 +1,129 @@
+  #     - 'main'
+  # pull_request:
+  #   branches:
+  #     - '**'
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04  
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)